deepchem/models/torch_models/gnn.py

```diff
@@ -178,6 +180,36 @@ def forward(self, data):
         return out
 
 
+class Discriminator(nn.Module):
+    """
+    This discriminator module is a linear layer without bias, used to measure the similarity between local node representations (`x`) and global graph representations (`summary`).
+
+    The goal of the discriminator is to distinguish between positive and negative pairs of local and global representations.
+    """
+    def __init__(self, hidden_dim):
```

Comment:
Can you document the parameter here in the numpydoc style?

Reply:
done